### PR TITLE
Toolbar with Breadcrumbs, improved Start/Stop and more

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- [x] New toolbar in both modes (workbench and guided mode). Contains:
  - [x] Breadcrumb
  - [x] Node Status icon in breadcrumb + tooltip (only guided)
  - [x] Prev/Next buttons (only guided)
  - [x] Run/Stop buttons
    - [x] Different options for Run Button
      - "Run"
      - "Run Node(s)" + "Run All"

#### Bonus
- [x] Quality can be viewed/updated by clicking the stars on card
- [x] SVG layer initialization improved

New toolbar:
toolbar

Stop/Run (workbench):
runStop

Stop/Run (guided):
runStop2

Quality by clicking the stars on card:
quality

## Related issue/s
related to https://github.com/ITISFoundation/osparc-issues/issues/327

### CODE REVIEW

**Review by KZzizzle — APPROVED** [review]
go go go! (NICE)

File: services/web/client/source/class/osparc/desktop/SlideShowToolbar.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    // overriden
+    _populateGuidedNodesLayout: function() {
+      const study = this.getStudy();
+      if (study) {
+        const slideShow = study.getUi().getSlideshow();
+        const nodes = [];
+        for (let nodeId in slideShow) {
+          const node = slideShow[nodeId];
+          nodes.push({
+            ...node,
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/desktop/SlideShowToolbar.js:58]: what are those ..., for?

File: services/web/client/source/class/osparc/desktop/SlideShowToolbar.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            nodeId
+          });
+        }
+        nodes.sort((a, b) => (a.position > b.position) ? 1 : -1);
+        const nodeIds = [];
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/desktop/SlideShowToolbar.js:63]: strange language that defines a ``const`` can change it :-) I guess const prevent assignment but not chancing the content of the container. ... still strange :-)

**Review by pcrespov — APPROVED** [review]
Every iteration looks better and better
I enjoy a lot seeing how much data is getting displayed and arranged in the UI
well done

PS: i cannot really judge/review all the code and leave it for more capable reviewers

File: services/web/client/source/class/osparc/desktop/StudyEditor.js [hunk 3]
```diff
@@ -135,6 +154,153 @@ qx.Class.define("osparc.desktop.StudyEditor", {
...
+        } else {
+          this.getLogger().error(null, "Failed submitting pipeline");
+        }
+        startStopButtonsWB.setRunning(false);
+        startStopButtonsSS.setRunning(false);
+      }, this);
+      if (selectedNodeIDs.length) {
+        req.setRequestData({
+          "subgraph": selectedNodeIDs
+        });
```
Comment (sanderegg) [inline services/web/client/source/class/osparc/desktop/StudyEditor.js:220]: you can add the force_restart field here.. once https://github.com/ITISFoundation/osparc-simcore/pull/2094 goes in

**Review by sanderegg — APPROVED** [review]
pretty nice!!

File: services/web/client/source/class/osparc/desktop/SlideShowToolbar.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/SlideShowToolbar.js:58]: [That's the object spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)

File: services/web/client/source/class/osparc/desktop/SlideShowToolbar.js (hunk 2 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/SlideShowToolbar.js:63]: It is a ```const``` in the sense that can't be reassigned, but it is [mutable](https://developer.mozilla.org/en-US/docs/Glossary/Mutable):
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

File: services/web/client/source/class/osparc/desktop/StudyEditor.js (hunk 3 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/StudyEditor.js:220]: Exactly 💪

**Pull request merged**